homeassistant/components/history/__init__.py

```diff
@@ -700,6 +700,8 @@ def __init__(self, row):  # pylint: disable=super-init-not-called
         self._row = row
         self.entity_id = self._row.entity_id
         self.state = self._row.state
+        if self.state is None:
+            self.state = ""
```

Comment:
Since it's always None or a string, this could work too.

```suggestion
        self.state = self._row.state or ""
```

Guess it all doesn't really matter 😄

Reply:
That is a sexy construct but still feels weird to me. Very Pythonic I would say.
I like how assignments in expressions in Python caused so much fuzz... while this, essentially is one 😄

Reply:
I was worried about the `0` case but its actually going to be `"0"` so `or` works better 👍